packages/react-components/react-skeleton/src/components/SkeletonItem/SkeletonItem.types.ts

```diff
@@ -0,0 +1,51 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type SkeletonItemSlots = {
+  root: Slot<'div'>;
+};
+
+/**
+ * Sizes for the SkeletonItem
+ */
+export type SkeletonItemSize = 8 | 12 | 16 | 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
```

Comment:
Design should follow up to change this out for tokens.  The specific sizes are assumed by callers to be pixels and we did a bunch of removing unions like these from other components.